### ISSUE TITLE
Pressing the `space` key doesn't execute the `insertTable` command

### ISSUE THREAD

**Issue by Dumluregn** [ckeditor/ckeditor5#12344]
## 📝 Provide detailed reproduction steps (if any)

1.  Go to a demo with table, e.g. https://ckeditor5.github.io/docs/nightly/ckeditor5/latest/examples/builds-custom/full-featured-editor.html.
2.  Open an insert table toolbar.
3.  Press `space`.

### ✔️ Expected result

Table should be inserted.

### ❌ Actual result

Command is not executed.

### ❓ Possible solution

`Space` should probably just get the same listener as `enter` key.

## 📃 Other details

*   Browser: all
*   OS: all
*   First affected CKEditor version: the one introducing grid keyboard handling (probably 35.1.0)
*   Installed CKEditor plugins: table

---

If you'd like to see this fixed sooner, add a 👍 reaction to this post.

**Comment by oleq** [comment]
To address this, we could either

*   Change `div` to `button` around here [https://github.com/ckeditor/ckeditor5/blob/7dfb504a3339fe287627bf422cae8c2a295303bf/packages/ckeditor5-table/src/ui/inserttableview.js#L268](https://github.com/ckeditor/ckeditor5/blob/7dfb504a3339fe287627bf422cae8c2a295303bf/packages/ckeditor5-table/src/ui/inserttableview.js#L268)
*   Make them all instances of `ButtonView`. This could be quite heavy but OTOH the entire grid is lazy loaded.

**Comment by mlewand** [comment]
To sum it up let's try first with ButtonView approach. Just test it out with some not-so-great workstation (mb air, slightly older pc, whatever).

If it performs well then let's go with ButtonView implementation.

If it shows slowdown (similar like special characters) let's fallback to alternative implementation, just a simple listener.
